src/Models/NonhydrostaticModels/update_hydrostatic_pressure.jl

```diff
@@ -3,17 +3,17 @@ using Oceananigans.ImmersedBoundaries: PartialCellBottom, ImmersedBoundaryGrid
 
 """
 Update the hydrostatic pressure perturbation pHY′. This is done by integrating
-the `buoyancy_perturbation` downwards:
+the `buoyancy_perturbationᶜᶜᶜ` downwards:
 
-    `pHY′ = ∫ buoyancy_perturbation dz` from `z=0` down to `z=-Lz`
+    `pHY′ = ∫ buoyancy_perturbationᶜᶜᶜ dz` from `z=0` down to `z=-Lz`
```

Comment:
ok, is this really what's happening?

$$ \int_0^{-L_z} p' \mathrm{d}z $$

Seems a bit odd... why don't we flip the limits and add a minus sign?

Reply:
no, its

$$ \int^0_z p' \text{ d}z $$

Reply:
"down to" is supposed to mean the summation occurs in the -z direction

Reply:
You could instead say "we sum from z to 0". But that doesn't explain why the code says `for k = grid.Nz-1:-1:1`